release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
+
+{product-title} release 4.7.18 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:2502[RHBA-2021:2502] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:R2503[RHBA-2021:2503] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6139672[{product-title} 4.7.18 container image list]
+
+[id="ocp-4-7-18-features"]
+==== Features
+
+[id="ocp-4-7-18-builds-telemetry-metric"]
+===== New Telemetry metric for number of builds by strategy
+In this update, Telemetry includes a new `openshift:build_by_strategy:sum` gauge metric, which sends the number of builds by strategy type to the Telemeter Client. This metric gives site reliability engineers (SREs) and product managers visibility into the kinds of builds that run on {product-title} clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1969963[*BZ#196963*])
+
+[id="ocp-4-7-18-bug-fixes"]
+==== Bug fixes
+
+* Previously, if you created a pipeline with parameters that had empty strings (`”`) as the default value, the fields in the {product-title} web console did not accept the empty strings. The current release fixes this issue. Now, `”` is supported as a valid default property within the parameters section. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966275[*BZ#1966275*])
+
+* Previously, the Cluster Version Operator reported a `ClusterOperatorDegraded` alert when ClusterOperator resources were degraded for ten minutes, and this alert sometimes fired prematurely during installation as resources were still being created. This update changes the 10-minute period to 30 minutes, providing enough time for the installation to progress without premature `ClusterOperatorDegraded` alerts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1957991[*BZ#1957991*])
+
+* When installing to vSphere, the bootstrap machine sometimes did not correctly update the nameservers in the `/etc/resolv.conf` file. As a result, the bootstrap machine couldn't access the temporary control plane, and installations failed. This fix includes changes that makes finding the right line to update more reliable. With the bootstrap manager now able to access its temporary control plane, installations are able to succeed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1967355[*BZ#1967355*])
+
+* Before this update, the Pipeline ServiceAccount did not use secrets created during the git import flow for private git repositories, causing these Pipelines to fail. This update fixes the problem by adding annotations to the secrets and to the Pipeline ServiceAccount. Pipelines for private git repositories now run correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970485[*BZ#1970485*])
```

Comment:
Use either:

```
`ServiceAcount` resource
```
or
```
service account
```

x2

Reply:
See [RH Supplementary Guide](https://redhat-documentation.github.io/supplementary-style-guide/#git) on usage for "Git" vs `git`.

Reply:
Agreed! Changed!